price-feeder/cmd/price-feeder.go

```diff
@@ -165,10 +202,14 @@ func startPriceFeeder(ctx context.Context, cfg config.Config, oracle *oracle.Ora
 	}
 }
 
-func startPriceOracle(ctx context.Context, oracle *oracle.Oracle) error {
+func startPriceOracle(ctx context.Context, oracle *oracle.Oracle, cancel context.CancelFunc) error {
```

Comment:
`startPriceOracle` should not take a `CancelFunc`. It was originally just a stub, but `startPriceOracle` should look more like `startPriceFeeder`.

i.e.

```go
// ...

go func() {
    log.Info().Msg("starting price-feeder oracle...")
    srvErrCh <- oracle.Start(ctx)
}

// ...
for {
  select {
    case <-ctx.Done():
      // ...

    case err := <-srvErrCh:
      // ...
  }
}
```

Reply:
Ty, didn't know the appropriate way to handle with goroutines